conda/cli/main_list.py

```diff
@@ -236,9 +244,10 @@ def print_packages(
     return exitcode
 
 
-def print_explicit(prefix, add_md5=False):
+def print_explicit(prefix, add_md5=False, with_auth=False):
```

Comment:
I like the imperative name better for this new argument, since it makes clear that the function _does_ process the URL and doesn't add authentication as implied with "with"

```suggestion
def print_explicit(prefix, add_md5=False, remove_auth=True):
```